stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.2.0
+version: 7.2.1
```

Comment:
Any change introducing a new helm-value must bump a minor version, for example: `7.3.0`.
Therefore, you will also need to modify the `README.md` and `UPGRADE.md`.

Reply:
The `README.md` lists `flower.service.*` and says `<see values.yaml>`, in order to add the new value I would need to document all 7 values that already exist because the new value is added to that same section, and then if I were to do that we might as well add all 9 `web.service` values in place of `web.service.*`, but that seems excessive. So for now I just updated `UPGRADE.md`. Let me know if we should document all `service` values in both sections but I would likely need some help in documenting the existing values

Reply:
I was meaning adding a link to the new `UPGRADE.md` (at the top of the `README.md`)

Reply:
Oh yes, forgot that part, doing that now

Reply:
ok, updated